cypress/obj/loginPage.js

```diff
@@ -4,6 +4,12 @@ class LoginPage
     {
         cy.visit('/');
     }
+
+    visitSpecificURL(url)
```

Comment:
The way you wrote this function makes it common, and you could extract this to some common.js file and user it from everywhere :)